trussworks/react-uswds
# feat: New Component SummaryBox

# Summary

This PR adds the [USWDS SummaryBox component](https://designsystem.digital.gov/components/summary-box/). 

## Related Issues or PRs

closes #988 

I took a different approach than the story outlined, as suggested by Hana, that prioritizes flexibility for the user over providing more structure. 

If in the future we wish to constrain the component further, we could extend this component using the proposed implementation from the issue. To do so, we would add the `<ul>` into SummaryBox, and create two subcomponents that would provide the rest of the skeleton for the list items and links; `SummaryBoxInfoItem` with a `<li>` root element, and `SummaryBoxLink` which could be implemented using the existing `<Link asCustom />` component, following the pattern in `BreadcrumbLink` and `IdentifierLink`.


## How To Test

Check out this branch and run `yarn test` to execute tests and `yarn storybook` to see the new component in action. 

Alternatively, you can scroll to the bottom of this page and click "Show environments" on the right above the comment input box.
 

### Screenshots
![Screen Shot 2021-04-05 at 6 17 28 PM](https://user-images.githubusercontent.com/16230705/113645170-3b79b180-963b-11eb-8b5f-901467b83e19.png)
